### PR TITLE
Initial HTML Report Class

### PR DESCRIPTION
**Pull request by WillAllmeida**
Create the HTML Report Class with initial handlers. Chart handlers need to be finished.

Related to #11

### CODE REVIEW

File: experiments/htmlrender.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+st.header("This is a header")
+st.info("This is info")
+st.subheader("This is a subheader")
+st.button("This is a button")
+df = pd.DataFrame(np.random.randn(10, 5), columns=("col %d" % i for i in range(5)))
+
+st.table(df)
+st.dataframe(df, 100, 250)
+
+## line chart
```
Comment (sammyzord) [inline experiments/htmlrender.py]: Please fix comments to adhere to pycodestyle stipulations
```suggestion
# line chart
```

File: experiments/htmlrender.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+chart_data = pd.DataFrame(
+np.random.randn(30, 3),
+columns=['a', 'b', 'c'])
+st.line_chart(chart_data)
+
+## area chart
```
Comment (sammyzord) [inline experiments/htmlrender.py]: Please fix comments to adhere to pycodestyle stipulations
```suggestion
# area chart
```

File: experiments/htmlrender.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+chart_data = pd.DataFrame(
+np.random.randn(40, 3),
+columns=['a', 'b', 'c'])
+st.area_chart(chart_data)
+
+## bar chart
```
Comment (sammyzord) [inline experiments/htmlrender.py]: Please fix comments to adhere to pycodestyle stipulations
```suggestion
# bar chart
```

**Review by sammyzord — CHANGES_REQUESTED** [review]
Please fix linting issues

**Comment by WillAllmeida** [comment]
A commit was published to correct linting issues

**Review by sammyzord — APPROVED** [review]

**Pull request merged**